test.js

```diff
@@ -50,6 +50,39 @@ test('register', function (t) {
     })
   })
 
+  t.test('Expose jwt methods namespaced', function (t) {
+    t.plan(14)
+
+    const fastify = Fastify()
+    fastify.register(jwt, { secret: 'test', namespace: 'security', jwtVerify: 'securityVerify', jwtSign: 'securitySign' })
+    fastify.register(jwt, { secret: 'hello', namespace: 'secure', jwtVerify: 'secureVerify', jwtSign: 'secureSign' })
```

Comment:
Could you add a 401 test too with these custom namespaces?
It is not clear to me what will happen: should the user trigger one token that verify to two authorization?

Could you update the readme accordingly?

Reply:
@Eomm Can you explain your question in more detail? I've added some documentation as requested.

Reply:
Nevermind, I was thinking of a bad usage of this feature